src/Date.re

```diff
@@ -1,7 +1,26 @@
 let nullable = Faker.nullable;
 
+type dateOption = {
+  .
+  "abbr": bool,
+  "context": bool,
+};
+
 [@bs.deriving abstract]
-type t = {past: (Js.Nullable.t(int), Js.Nullable.t(Js.Date.t)) => Js.Date.t};
+type t = {
+  past: (Js.Nullable.t(int), Js.Nullable.t(Js.Date.t)) => Js.Date.t,
+  future: (Js.Nullable.t(int), Js.Nullable.t(Js.Date.t)) => Js.Date.t,
+  between: (Js.Date.t, Js.Date.t) => Js.Date.t,
+  recent: Js.Nullable.t(int) => Js.Date.t,
+  month: Js.Nullable.t(dateOption) => string,
+  weekday: Js.Nullable.t(dateOption) => string,
+};
 [@bs.module "faker"] external fakers : t = "date";
 let past = (~years=?, ~refDate=?, ()) =>
   past(fakers, nullable(years), nullable(refDate));
+let future = (~years=?, ~refDate=?, ()) =>
+  future(fakers, nullable(years), nullable(refDate));
+let between = (from, to_) => between(fakers, from, to_);
+let recent = (~days=?, ()) => recent(fakers, nullable(days));
+let month = (~opts=?, ()) => month(fakers, nullable(opts));
```

Comment:
what do you think about having the options spread out already? instead of having to write `~opts={"something": ..."}`?

I don't know if we can make `month` accept `{ . "abbr": nullable(bool), "context": nullable(bool) }` instead, and then it'll be easier to define that 😸